### DISCUSSION TITLE
Adding a new installation method to the README

### DISCUSSION THREAD

**Discussion by lunrenyi** [Byron/dua-cli#258] [Q&A] [answered]
- Hi, we have implemented a lightweight [package manager using shell and awk](https://www.x-cmd.com/pkg/).
It helps you download dua release packages from the internet and extract them into a unified directory for management, without requiring root permissions.

- **I mean, can the installation method provided by x-cmd be added to the dua README?**[The installation method for the x command](https://www.x-cmd.com/start/)
  ```sh
  x env use dua
  ```

Looking forward to your reply :) !

**Comment by Byron** [comment, marked as answer]
Yes, `x` looks great, please feel free to contribute the installation method.
